src/status_im2/contexts/quo_preview/tags/context_tags.cljs

```diff
@@ -51,73 +54,113 @@
   []
   (let [state (reagent/atom {:label        "Name"
                              :channel-name "Channel"
-                             :type         :group-avatar})]
+                             :type         :group-avatar
+                             :duration     "00:32"})]
     (fn []
-      (let [contacts {example-pk  {:public-key   example-pk
-                                   :primary-name "Automatic incompatible Coati"
-                                   :photo        example-photo}
-                      example-pk2 {:public-key   example-pk2
-                                   :primary-name "Clearcut Flickering Rattlesnake"
-                                   :photo        example-photo2}}
+      (let [contacts             {example-pk  {:public-key   example-pk
+                                               :primary-name "Automatic incompatible Coati"
+                                               :photo        example-photo}
+                                  example-pk2 {:public-key   example-pk2
+                                               :primary-name "Clearcut Flickering Rattlesnake"
+                                               :photo        example-photo2}}
             contacts-public-keys (map (fn [{:keys [public-key]}]
                                         {:key   public-key
                                          :value (get-in contacts [public-key :primary-name])})
                                       (vals contacts))
-            current-username (if (seq (:contact @state))
-                               (->> @state
-                                    :contact
-                                    contacts
-                                    multiaccounts/displayed-name)
-                               "Please select a user")
-            descriptor
-            (cond
```

Comment:
In the preview screen, the descriptor existed only for some types of context-tags, if we chose a type not contemplated, the descriptor menu disappeared making impossible to switch back to another type, now this is fixed